### ISSUE TITLE
Add support to launch on Android TV from home screen

### ISSUE THREAD

**Issue by KaptnJack** [ram-on/SkyTube#401]
It would be greate to add support to launch app from Android TV home page.
Need to add in AndroidManifest.xml:
<category android:name="android.intent.category.LEANBACK_LAUNCHER"/>
and add android:banner="@drawable/<app_logo.png>" in activity entry.
May be need some more actions.

**Comment by jordanade** [comment]
Yes, a full Android TV-oriented experience would be great! Especially if it includes populating a home-screen "channel" as the official app does.

**Comment by vebaev** [comment]
Is there any update on the matter?

**Comment by drogga** [comment]
Even I can (probably) make it show a tile in the leanback launcher, but it won't magically make it compatible, some things work by navigating with the remote's D-Pad (videos can't be selected with it), some with a virtual mouse cursor, but most things need a real mouse or a "air" one, of your remote supports it, but the original one that comes with Philips TV (with A.5.1.1) doesn't supports air gestures, so IDK if those work.
